src/utils/event-sourcing/src/event_store.rs

```diff
@@ -28,11 +28,7 @@ pub trait EventStore<Proj: Projection>: Send + Sync {
     /// The `query` argument must be the same as query passed when retrieving
     /// the events. It will be used prior to saving events to ensure that there
     /// were no concurrent updates that could've influenced this transaction.
-    async fn save_events(
-        &self,
-        query: &Proj::Query,
-        events: Vec<Proj::Event>,
-    ) -> Result<EventID, SaveEventsError>;
+    async fn save_events(&self, events: Vec<Proj::Event>) -> Result<EventID, SaveEventsError>;
```

Comment:
What's the reason for this change?

`query` is actually very important here as it will play the key role in optimistic concurrency control (see comment above).

Reply:
Restored. Nothing uses this, this causes confusion.